lib/subscription.js

```diff
@@ -3,7 +3,7 @@ const Websocket = require('fastify-websocket')
 const SubscriptionConnection = require('./subscription-connection')
 const GRAPHQL_WS = 'graphql-ws'
 
-export function handle (conn) {
+function handle (conn) {
```

Comment:
Good work on the creating a test for `handle`. Can you set change `conn.pipe(conn)` to `conn.end(JSON.stringify({ error: "unknown route" }))`, and update the test accordingly?